### PR TITLE
fix: less max splits and higher distribution percent for less routes zksync

### PR DESCRIPTION
**Pull request by jsy1218**

### CODE REVIEW

**Comment by jsy1218** [comment]
* **#745** <a href="https://app.graphite.dev/github/pr/Uniswap/routing-api/745?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @jsy1218 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by mikeki — APPROVED** [review]

**Review by zhongeric — APPROVED** [review]

File: lib/handlers/shared.ts [hunk 1]
```diff
@@ -93,8 +93,8 @@ export const DEFAULT_ROUTING_CONFIG_BY_CHAIN = (chainId: ChainId): AlphaRouterCo
         },
         maxSwapsPerPath: 3,
         minSplits: 1,
-        maxSplits: 7,
-        distributionPercent: 5,
+        maxSplits: 2,
+        distributionPercent: 25,
```
Comment (cgkol) [inline lib/handlers/shared.ts:97]: this pr now affects all chains (mainnet too). I think we want to case it for zksync only right? cc: @jsy1218 @zhongeric 

I was wondering why our latency wnet down too.. but this explains it

Comment (xrsv) [reply lib/handlers/shared.ts:97]: i was also checking the latency metrics today, and ended up to this PR as well.
Agree this needs to apply to zksync only

Comment (jsy1218) [reply lib/handlers/shared.ts:97]: fixed in https://github.com/Uniswap/routing-api/pull/746/files

**Pull request merged**